tests/acceptance/infinity-route-offset-trigger-test.js

```diff
@@ -0,0 +1,107 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from '../helpers/start-app';
+import Pretender from 'pretender';
+import faker from 'faker';
+
+var App, server;
+
+module('Acceptance: Infinity Route - offset trigger', {
+  setup() {
+    var posts = [];
+
+    for (var i = 0; i < 50; i++) {
+      posts.push({id: i, name: faker.company.companyName()});
+    }
+
+    App = startApp();
+    server = new Pretender(function() {
+      this.get('/posts', function(request) {
+        var body, subset, perPage, startPage, offset;
+
+        if (request.queryParams.category) {
+          subset = posts.filter(post => {
+            return post.category === request.queryParams.category;
+          });
+        } else {
+          subset = posts;
+        }
+        perPage = parseInt(request.queryParams.per_page, 10);
+        startPage = parseInt(request.queryParams.page, 10);
+
+        var pageCount = Math.ceil(subset.length / perPage);
+        offset = perPage * (startPage - 1);
+        subset = subset.slice(offset, offset + perPage);
+
+        body = { posts: subset, meta: { total_pages: pageCount } };
+
+        return [200, {"Content-Type": "application/json"}, JSON.stringify(body)];
+      });
+    });
+  },
+  teardown() {
+    Ember.run(App, 'destroy');
+    server.shutdown();
+  }
+});
+
+
+test('it should start loading more items when the scroll is on the very bottom ' +
+  'when triggerOffset is not set', assert => {
+  assert.expect(8);
+  visit('/test-scrollable');
+
+  andThen(() => {
+    var postList = find('ul');
+    var infinityLoader = find('.infinity-loader');
+
+    assert.equal(postList.find('li').length, 25, "Two items should be in the list");
+    assert.equal(infinityLoader.hasClass('reached-infinity'), false, "Infinity should not yet have been reached");
+    var triggerOffset = postList.get(0).scrollHeight - postList.height();
+    postList.scrollTop(triggerOffset - 20);
+    triggerEvent('ul', 'scroll');
```

Comment:
`triggerEvent` is an async test helper, just like `andThen` and `click`. It should be in the top level closure

see: http://guides.emberjs.com/v1.10.0/testing/test-helpers/#toc_asynchronous-helpers
```js
andThen(...);
triggerEvent(...);
click(...);
```